genfragments/ThirteenTeV/bbHtoZatoLLChiChi_2HDMa/bbH_Za_LLChiChi_MH800_Ma600_MChi45_4f-TuneCP5_13TeV-fragment.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
```

Comment:
Please delete the fragment files if you are using just a plain CP5 tune settings. No need to commit them with externalLHEproducers injected with every single gridpacks